.github/workflows/close-stale-feature-requests.yml

```diff
@@ -0,0 +1,43 @@
+name: Close stale feature requests
+on:
+  schedule:
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. We value your input
```

Comment:
Maybe it's just me, but "We value your input" in an automated message sounds like an empty platitude. (But if you or others disagree, let's keep it.)

Reply:
I remove dit.